README.md

```diff
@@ -20,8 +20,8 @@ Basic usage of this module is as follows:
 
 ```hcl
 module "cloud_functions2" {
-  source  = "terraform-google-modules/cloud-functions/google"
-  version = "~> 0.1.0"
+  source  = "GoogleCloudPlatform/cloud-functions/google"
+  version = "~> 0.1.1"
```

Comment:
Hmm, v0.1.1 won't yet be released.  Might consider pessimistic pinning to a minor version?

```suggestion
  version = "~> 0.1"
```

Reply:
v0.1.0 is already released isnt it? Since this is minor change, should I go with v0.2.0?

Reply:
Hi @prabhu34 - The `release-please` bot will automatically generate a PR for the next version PR based on commits.  If this is the only commit, it will likely be `0.2.0`.  However that version won't initially exist when this PR is committed, so my suggestion is to use the pessimistic operator `~> 0.1` which will currently match `0.1.1` and then `0.2.0` when released.  Cheers!